app/models.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from logging import Logger
+from typing import TYPE_CHECKING, Literal
+
+import yaml
+from checkov.kubernetes.runner import Runner as K8sRunner
+from checkov.main import Checkov
+from checkov.sca_image.runner import Runner as ScaImageRunner
+
+from app.consts import CHECKOV_CONFIG_PATH
+
+if TYPE_CHECKING:
+    from checkov.common.output.baseline import Baseline
+    from checkov.common.runners.runner_registry import RunnerRegistry
+    from flask import Flask
+
+
+class CheckovWhorf(Checkov):
```

Comment:
Should we place that class under the `models` file? I feel like it belongs on a separate file, since it's not a model...

Reply:
yeah, sounds good. previously I had in the `utils.py` because it didn't fit anywhere 😄 
@rotemavni what do you think about `checkov.py` or `checkov_whorf.py`?

Reply:
`checkov_whorf.py` is my winner